packages/cashier/src/Components/Error/error.jsx

```diff
@@ -33,7 +32,7 @@ const ErrorComponent = ({ header, message, button_link, onClickButton, button_te
     </div>
 );
 
-const Error = ({ error, setErrorMessage }) => {
+const Error = ({ error }) => {
```

Comment:
One of the best things about stores is that you don't have to pass store props to every component, you can call them directly from the store. I think this is coming from `withdrawal.jsx` and `error` has been added to the connect function there so instead of passing `error` to this component as a prop, why not add `error` to the connect function here?

Reply:
@carol-binary We are using `Error` component not only in withdraw but also in `paymentAgentList`, `accountTransfer`, `deposit` and `paymentAgentTransfer` and each of them has their own error object which needs to be passed from their component, and in those components I use connect for passing error.